core/src/io/anuke/mindustry/game/EventType.java

```diff
@@ -327,11 +327,40 @@ public PlayerConnect(Player player){
 
     public static class PlayerLeave{
         public final Player player;
-        
+
         public PlayerLeave(Player player){
             this.player = player;
         }
     }
-           
+    public static class PlayerBanEvent{
+        public final Player player;
+
+        public PlayerBanEvent(Player player) {
```

Comment:
Unnecessary space, use the formatting guidelines. Happens for every method here.